packages/components/package.json

```diff
@@ -52,8 +52,11 @@
 		"@wp-g2/utils": "^0.0.160",
 		"classnames": "^2.2.5",
 		"create-emotion": "^10.0.27",
+		"deepmerge": "^4.2.2",
 		"dom-scroll-into-view": "^1.2.1",
 		"downshift": "^6.0.15",
+		"emotion": "^11.0.0",
+		"fast-deep-equal": "^3.1.3",
 		"gradient-parser": "^0.1.5",
 		"highlight-words-core": "^1.2.2",
 		"hoist-non-react-statics": "^3.3.2",
```

Comment:
I wonder if we have alternatives for `fast-deep-equal` and `deep merge` in our codebase already? Maybe lodash alternatives?

Reply:
Updated to use `isEqual` and `merge` which are both recursive/deep functions.